src/components/organisms/playerScreenContent/PlayerScreenContent.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Col, Row, Typography } from 'antd';
+
+import Context from '../../../context';
+import styles from './playerScreenContent.module.scss';
+
+const PlayerScreenContent = ({ castDetails, directorDetails, story }) => {
```

Comment:
this component seems a duplicate of MovieDetailsContent component, please reuse the same and delete this one